<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>locAlgrithom</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="showPath">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>70</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>显示路径</string>
    </property>
   </widget>
   <widget class="QPushButton" name="beginTrack">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>100</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>track</string>
    </property>
   </widget>
   <widget class="uiCanvas" name="canvas" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>600</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
   </widget>
   <widget class="QPushButton" name="reset">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>100</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>reset</string>
    </property>
   </widget>
   <widget class="QPushButton" name="next">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>130</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>next</string>
    </property>
    <property name="shortcut">
     <string>Right</string>
    </property>
    <property name="autoRepeat">
     <bool>true</bool>
    </property>
    <property name="autoRepeatDelay">
     <number>250</number>
    </property>
    <property name="autoRepeatInterval">
     <number>10</number>
    </property>
    <property name="autoDefault">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="allPos">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>40</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>显示raw</string>
    </property>
   </widget>
   <widget class="QPushButton" name="previous">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>130</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>previous</string>
    </property>
    <property name="shortcut">
     <string>Left</string>
    </property>
    <property name="autoDefault">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>210</y>
      <width>151</width>
      <height>170</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout" columnstretch="1,8,7">
     <item row="4" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>3</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>4</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="raw_0">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="raw_1">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QLineEdit" name="refine_2">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="raw_3">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="raw_2">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLineEdit" name="refine_0">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLineEdit" name="refine_1">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QLineEdit" name="refine_3">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>1</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>2</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>raw</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>diff</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>5</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="raw_4">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="raw_5">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QLineEdit" name="refine_4">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="6" column="2">
      <widget class="QLineEdit" name="refine_5">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="showRadius">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>40</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>显示半径</string>
    </property>
   </widget>
   <widget class="QPushButton" name="showTrace">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>70</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>显示iter</string>
    </property>
   </widget>
   <widget class="uiUsrFrame" name="UsrFrm">
    <property name="geometry">
     <rect>
      <x>815</x>
      <y>390</y>
      <width>160</width>
      <height>180</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="distCountEdit">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>160</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="gotoCount">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>160</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>goto</string>
    </property>
    <property name="autoDefault">
     <bool>true</bool>
    </property>
   </widget>
   <zorder>showPath</zorder>
   <zorder>beginTrack</zorder>
   <zorder>reset</zorder>
   <zorder>next</zorder>
   <zorder>allPos</zorder>
   <zorder>previous</zorder>
   <zorder>gridLayoutWidget</zorder>
   <zorder>showRadius</zorder>
   <zorder>showTrace</zorder>
   <zorder>UsrFrm</zorder>
   <zorder>canvas</zorder>
   <zorder>distCountEdit</zorder>
   <zorder>gotoCount</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>位置算法</string>
    </property>
    <addaction name="actionFullCentroid"/>
    <addaction name="actionSubLS"/>
    <addaction name="actionTwoCenter"/>
    <addaction name="actionTaylorSeries"/>
    <addaction name="actionWeightedTaylor"/>
    <addaction name="separator"/>
    <addaction name="actionKalmanTaylor"/>
    <addaction name="actionLMedS"/>
   </widget>
   <widget class="QMenu" name="menuview">
    <property name="title">
     <string>视图</string>
    </property>
    <addaction name="actionZoomIn"/>
    <addaction name="actionZoomOut"/>
    <addaction name="separator"/>
    <addaction name="actionCapPix"/>
   </widget>
   <widget class="QMenu" name="menuNLOS">
    <property name="title">
     <string>NLOS检测&amp;滤波</string>
    </property>
    <addaction name="actionWylie"/>
    <addaction name="actionMultiPoint"/>
    <addaction name="separator"/>
    <addaction name="actionRes"/>
    <addaction name="actionSumDist"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>标签跟踪</string>
    </property>
    <addaction name="actionKalmanTrack"/>
    <addaction name="actionKalmanLiteTrack"/>
    <addaction name="actionKalmanInfoTrack"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actionRead_ini"/>
    <addaction name="actionRead_dist"/>
    <addaction name="actionRead_dist_2"/>
    <addaction name="actionRead_picture"/>
   </widget>
   <widget class="QMenu" name="menuARM">
    <property name="title">
     <string>ARM</string>
    </property>
    <addaction name="actioncalcTagPos_ARM"/>
   </widget>
   <addaction name="menu_3"/>
   <addaction name="menuNLOS"/>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menuview"/>
   <addaction name="menuARM"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionFullCentroid">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>全质心</string>
   </property>
  </action>
  <action name="actionSubLS">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>相减最小二乘</string>
   </property>
  </action>
  <action name="actionZoomIn">
   <property name="text">
    <string>Zoom In</string>
   </property>
  </action>
  <action name="actionZoomOut">
   <property name="text">
    <string>Zoom Out</string>
   </property>
  </action>
  <action name="actionTwoCenter">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>纯二维求中心</string>
   </property>
  </action>
  <action name="actionWylie">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Wylie算法</string>
   </property>
  </action>
  <action name="actionMultiPoint">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>多点比较</string>
   </property>
  </action>
  <action name="actionRes">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>残差比较</string>
   </property>
  </action>
  <action name="actionSumDist">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>距离和比较</string>
   </property>
  </action>
  <action name="actionKalmanTrack">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>kalman</string>
   </property>
  </action>
  <action name="actionKalmanLiteTrack">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>kalmanLite</string>
   </property>
  </action>
  <action name="actionRead_ini">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Read ini</string>
   </property>
  </action>
  <action name="actionRead_dist">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Read distance</string>
   </property>
  </action>
  <action name="actionRead_picture">
   <property name="text">
    <string>Read picture</string>
   </property>
  </action>
  <action name="actionRead_dist_2">
   <property name="text">
    <string>Read distance_2</string>
   </property>
  </action>
  <action name="actionTaylorSeries">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>TaylorSeries</string>
   </property>
  </action>
  <action name="actionKalmanInfoTrack">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>kalmanInfo</string>
   </property>
  </action>
  <action name="actionWeightedTaylor">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>WeightedTaylor</string>
   </property>
  </action>
  <action name="actionKalmanTaylor">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>KalmanTaylor</string>
   </property>
  </action>
  <action name="actioncalcTagPos_ARM">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>calcTagPos_ARM</string>
   </property>
  </action>
  <action name="actionLMedS">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>LMedS</string>
   </property>
  </action>
  <action name="actionCapPix">
   <property name="text">
    <string>截图</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>uiCanvas</class>
   <extends>QWidget</extends>
   <header>uiCanvas.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>uiUsrFrame</class>
   <extends>QFrame</extends>
   <header>uiUsrFrame.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>previous</tabstop>
  <tabstop>next</tabstop>
  <tabstop>distCountEdit</tabstop>
  <tabstop>gotoCount</tabstop>
  <tabstop>showTrace</tabstop>
  <tabstop>showPath</tabstop>
  <tabstop>showRadius</tabstop>
  <tabstop>reset</tabstop>
  <tabstop>raw_3</tabstop>
  <tabstop>raw_2</tabstop>
  <tabstop>raw_0</tabstop>
  <tabstop>refine_0</tabstop>
  <tabstop>refine_1</tabstop>
  <tabstop>refine_3</tabstop>
  <tabstop>raw_4</tabstop>
  <tabstop>raw_5</tabstop>
  <tabstop>refine_4</tabstop>
  <tabstop>refine_5</tabstop>
  <tabstop>raw_1</tabstop>
  <tabstop>beginTrack</tabstop>
  <tabstop>refine_2</tabstop>
  <tabstop>allPos</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
